src/cli_helper/config.yaml

```diff
@@ -56,6 +56,15 @@ subcommands:
       templates themselves.
     action: Commands::Configure::Rerender
 
+  configure_self: &configure_self
+    syntax: metal configure self [options]
+    summary: Configure the master Metalware node
+    description: >
+      Configures the node that metalware is running on. This is treated as the
+      metalware master node. This must be ran to configure metalware to match
+      the system environment. This command is intended to only be ran once.
```

Comment:
> This command is intended to only be ran once.

Is there any negative effect if it is run more than once?

Reply:
Probably, their are a whole bunch of bash scripts that run which probably can't be reran. But it is a repo, kb issue.

Reply:
OK - I guess we don't need to do anything special then to stop someone shooting themselves in the foot due to a bad repo, especially since this is just one of many ways they could do this. This LGTM then once #178 is merged.